components/calendar/calendar.js

```diff
@@ -163,6 +163,7 @@ class Calendar extends LocalizeStaticMixin(LitElement) {
 
 	static get properties() {
 		return {
+			dialog: { type: Boolean },
```

Comment:
Needed for accessibility, in order to add necessary attributes to wrap the calendar.

Reply:
Could we maybe hide this a bit more? Or could we detect it automatically somehow by seeing if it's inside a dropdown? Just don't want others to see it and set it if we can avoid it.

Reply:
Agreed - I like the idea of automatic detection. I'll look into that.